### PR TITLE
Template destination file permissions.

### PR DESCRIPTION
**Pull request by dadgar**
This PR allows setting the file permissions of the rendered template.

### CODE REVIEW

**Comment by jippi** [comment]
nice

would it be possible to get for artifacts too?

**Comment by dadgar** [comment]
The owner/group is harder since Nomad may not have that capability. Needs CAP_CHOWN which root has but other users don't by default

**Comment by dadgar** [comment]
Why do you need that?

File: website/source/docs/job-specification/template.html.md [hunk 1]
```diff
@@ -58,12 +65,8 @@ README][ct].
 - `destination` `(string: <required>)` - Specifies the location where the
   resulting template should be rendered, relative to the task directory.
 
-- `change_mode` `(string: "restart")` - Specifies the behavior Nomad should take
-  if the rendered template changes. The possible values are:
-
-  - `"noop"` - take no action (continue running the task)
-  - `"restart"` - restart the task
-  - `"signal"` - send a configurable signal to the task
+- `perms` `(string: "666")` - Specifies the rendered templates permissions. File
```
Comment (diptanu) [inline website/source/docs/job-specification/template.html.md]: template's

**Pull request merged**